src/Command/HttpClientAsync.php

```diff
@@ -35,45 +35,33 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $delays = [1, 3, 3, 5, 10];
+        // this command runs requests concurrently and read them in *network* order in 23s
+
+        $delays = [10, 3, 3, 5, 1];
         $responses = [];
 
         foreach ($delays as $offset => $delay) {
-            try {
-                $responses[] = $this->httpClient->request(
```

Comment:
the request method is lazy: it should do no network, no need to wrap it in a try/catch